CondFormats/L1TObjects/interface/CaloParams.h

```diff
@@ -128,9 +128,13 @@ namespace l1t {
       // veto region is seed tower +/- <=egIsoVetoNrTowersPhi
       unsigned isoVetoNrTowersPhi_;
 
+      // turn on/off EG ID cuts
```

Comment:
Hi @davidlange6 .  The short answer is No.  I am sorry about that.  

As commented here https://github.com/cms-sw/cmssw/pull/16318#issuecomment-255554079, the Stage1 workflows fail do to `L1TStage1Layer2Producer`. It uses the record `L1TCaloParamsRcd`, which turns out to have a mismatch with what is present in the GlobalTag. 

@davidlange6 , what is the suggestion?  Can we ask @ggovi for an update in GT?